resources/graphics.py

```diff
@@ -371,6 +371,10 @@
 				"setMemberHighlighted",
 				"setMemberFaded",
 				"setMemberFadedHighlighted",
+                "boxBlocker",
+                "sphereBlocker",
+                "planeBlocker",
+                "cylinderBlocker",
```

Comment:
We've ended up with spaces here rather than tabs.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/5895/commits/d628c7c96b4a18540586098a0d2aaedba6416ac4